.github/dependabot.yml

```diff
@@ -1,8 +1,7 @@
 version: 2
 updates:
-  - package-ecosystem: "pip" # See documentation for possible values
-    directory: "/" # Location of package manifests
-    insecure-external-code-execution: allow
+  - package-ecosystem: "pip"
+    directory: "/requirements"
     schedule:
```

Comment:
Why is the inseure-external-code-execution no longer allowed?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#insecure-external-code-execution

Reply:
I decided to remove this line after noticing it was not included in [ansys-templates] and was not causing any issues.

[ansys-templates]: https://github.com/pyansys/ansys-templates